src/app/src/components/FacilityListItemsTableRow.jsx

```diff
@@ -64,14 +81,20 @@ const FacilityListItemsTableRow = memo(({
             padding="none"
             style={listTableCellStyles.statusCellStyles}
         >
-            {status}
+            {
+                newFacility
+                    ? 'NEW_FACILITY'
+                    : status
+            }
         </TableCell>
     </TableRow>
 ), propsAreEqual);
 
 FacilityListItemsTableRow.defaultProps = {
     hover: false,
     handleSelectRow: null,
+    newFacility: false,
```

Comment:
Is `{ newFacility: true, oarID: null }` a legitimate possibility? If not, we can probably just use `oarID` as the only field, with `oarID === null` implying `newFacility === true`.

Reply:
If it does happen it would be due to a bug in the API, but I split these props to ensure that the view would be resilient to having a small problem in the data.

However, presumably every facility should always have an OAR ID.

Reply:
> However, presumably every facility should always have an OAR ID.

Is there any way to enforce that?

Reply:
I suppose it's conceivably enforced by being the primary key of the `Facility` model:

https://github.com/open-apparel-registry/open-apparel-registry/blob/develop/src/django/api/models.py#L321

The rhyming prop type for the list items marks it as `isRequired` but that's only checked at runtime: https://github.com/open-apparel-registry/open-apparel-registry/blob/develop/src/app/src/util/propTypes.js#L81

I guess I'd imagine that the API would 500 on the serializer crashing here, too:

https://github.com/open-apparel-registry/open-apparel-registry/blob/develop/src/django/api/serializers.py#L177

Reply:
Given the above enforcement, should we still keep the two fields? On one hand, it slightly improves readability. On the other, it implies a connection that could be missed when touching this code in the future.

Reply:
Happy to do either. My intention here was to separate out two things:

- we only want the view to create a link to the facility when it's both a new facility and there's an OAR ID to link to
- we only want to display NEW_FACILITY status when it's a new facility (but it doesn't matter if there's an OAR ID)

Currently both of these will be in place, so we could rely on both of them being there, but fusing them would overload what OAR ID is supposed indicate for the view and also couple the view more closely to the API, which is I think what informed deciding to split them.

However, on balance it's not a huge deal either way.

Reply:
Ah that makes sense. By having a separate field we ensure that if OAR ID is changed in the future, this behavior won't be affected. Let's keep the split then.